runtime/centrifuge/src/lib.rs

```diff
@@ -1573,7 +1573,7 @@ impl pallet_loans_ref::Config for Runtime {
 	type NonFungible = Uniques;
 	type Permissions = Permissions;
 	type Pool = PoolSystem;
-	type PriceId = PriceId;
+	type PriceId = u32;
```

Comment:
Why make it a raw type again?

Reply:
Our real type is `OracleKey`. It doesn't matter what you put as type there in altair & centrifuge runtimes because the price feature is not allowed as long as `NoPriceRegistry` type is attached to the loan pallet configuration.

Basically, I removed the `PriceId` from `cfg-primitives` and I placed an `u32` as a placeholder.